src/model.coffee

```diff
@@ -169,6 +169,24 @@ class Model extends Backbone.Model
       )
       .promise(options.returnValues.jqXhr)
 
+  destroy: (options = {}) ->
+    cleanUpAssociatedReferences = =>
+      for collectionName, collection of @storageManager.collections
+        for reference, modelName of collection.modelKlass.associations
+          associationKey = collection.modelKlass.associationDetails(reference).key
+          if _.isArray(modelName) && _.find(modelName, (m) => inflection.singularize(m) == @className())
```

Comment:
Yeah, we struggled with this because it can be something like `'tasks'` or `['tasks']`. We decided to go with `associator`, since that's what other code in this file referred to it as.